encfs/FileUtils.cpp

```diff
@@ -1160,10 +1171,10 @@ RootPtr createV6Config(EncFS_Context *ctx,
 
   // get user key and use it to encode volume key
   CipherKey userKey;
-  VLOG(1) << "useStdin: " << useStdin;
-  if (useStdin) {
+  VLOG(1) << "passSrc: " << opts->passSrc;
+  if (opts->passSrc != Pass_Prompt) {
```

Comment:
You're right -- on Windows we don't have `passwordProgram`, so I didn't see that this will never be executed with the current layout on Linux.  

We want to fall into `userKey = config->getUserKey(opts);` for either Pass_Stdin or Pass_Cmd.  What do you think about changing this if-statement to: 

```C++
if (opts->passSrc == Pass_Cmd || opts->passSrc == Pass_Stdin) {
```